packages/network/src/logic/Tracker.ts

```diff
@@ -67,6 +68,7 @@ export class Tracker extends EventEmitter {
         this.overlayConnectionRtts = {}
         this.locationManager = new LocationManager()
         this.instructionCounter = new InstructionCounter()
+        this.extraMetadatas = Object.create(null)
```

Comment:
`Object.create()` is preferable to `{}` due to less surprises.

E.g. with literal
```ts
const o1 = {}
o1['toString'] !== undefined // true

const o2 = Object.create(null)
o1['toString'] === undefined // true
```